docs/pages/workflow/android-studio-emulator.md

```diff
@@ -16,11 +16,9 @@ If you don't have an Android device available to test with, we recommend using t
 
 ![Android SDK location](/static/images/android-studio-sdk-location.png)
 
-- If you are on macOS or Linux, add the Android SDK location to your PATH using `~/.bash_profile` or `~/.bashrc`. You can do this by adding a line like `export ANDROID_SDK=/Users/myuser/Library/Android/sdk`.
+- If you are on macOS or Linux, add the Android SDK location to your PATH in `~/.bash_profile` or `~/.zshrc` by running this command: ``echo "export ANDROID_SDK=$HOME/Library/Android/sdk" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'` ``
```

Comment:
I think we should leave the original message in so it's clear to non-beginners what they need to do. so, something like:

> - If you are on macOS or Linux, add the Android SDK location to your PATH using `~/.bash_profile` or `~/.bashrc`. You can do this by adding a line like `export ANDROID_SDK=/Users/myuser/Library/Android/sdk`. If you prefer, you can copy and paste this command to handle this for you: ``echo "export ANDROID_SDK=$HOME/Library/Android/sdk" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'` ``

Reply:
Ok, that sounds good to me. I'll reword a bit and try some things out.

Reply:
Ok, I've updated in [`da97865` (#10988)](https://github.com/expo/expo/pull/10988/commits/da9786506f03a0e0522f1d938e9405b72fccd2e3), what do you think?

Reply:
The reason I got rid of `.bashrc` is because of this:

> This means that .bash_profile is great for commands that should run only once and .bashrc for commands that should run every time you start a new shell.
> 
> For example, PATH customization should only happen once

https://github.com/thoughtbot/til/blob/master/bash/bash_profile_vs_bashrc.md

Reply:
And used `.zshenv` because that's what it's meant for: 

> `.zshenv`: This is read first and read every time. This is where you set environment variables.

https://apple.stackexchange.com/a/388623/36202